x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/original_scenarios.md

```diff
@@ -0,0 +1,264 @@
+
+Examples:
+  | action                |
+  | installing all        |
+  | installing selected   |
+  | installing individual |
+  | updating all          |
+  | updating selected     |
+  | updating individual   |
+```
+
+#### **Scenario: No callout messages are displayed when rule package installation fails and no rules are avialble for install/update**
+
+```Gherkin
+Given user navigates to Rules Management Page
+And user is running a fresh instance
+And rule package installation fails
+Then no callouts message should be displayed
+And the number of available rules to install and upgrade in the badges does not change
+```
+
+### Add Elastic Rules Page
+
+#### **Scenario: New workflow elements are displayed on Rules Management Page**
+
+```Gherkin
+Given a user that doesn’t have `security_detection_engine` package installed
+When user is on Rules Management Page
+Then "+Add Elastic rules" menu with available Rules counter is displayed
+And Rule Updates tab is displayed
+And "+Add Elastic rules" button is displayed on empty Rules Table
+```
+
+#### **Scenario: Rules settings persist on Add Elastic Rules table**
+
+```Gherkin
+Given a user has Rules listed on Add Elastic Rules page
+When <case>
+Then the available rules state should persist for all the rules
+
+Examples:
+  | case                              |
+  | user reloads the page             |
+  | after switching table pagination  |
+  | after filtering and clear filters |
+```
+
+#### **Scenario: User can navigate back to Rules Management page**
+
+```Gherkin
+Given a user is on Add Rules Page
+When user navigates back to Rules Management page
+Then Rules Management Page is properly displayed
+```
+
+#### **Scenario: User can filter prebuilt rules by rule name or by tag**
+
+```Gherkin
+Given a user is on Add Rules Page
+When user filters by <filter>
+Then Add Rules Table is properly updated
+
+Examples:
+  | filter                  |
+  | rule name on search bar |
+  | Tag filter              |
+```
+
+### Rule Updates tab
+
+#### **Scenario: Rules settings persist on Rule Updates table**
+
+```Gherkin
+Given a user has Rules listed on Rule Updates table
+When <case>
+Then the rules with available updates state should persist
+
+Examples:
+  | case                              |
+  | user reloads the page             |
+  | after switching table pagination  |
+  | after filtering and clear filters |
+```
+
+#### **Scenario: User can navigate back to Rules Management tab**
+
+```Gherkin
+Given a user is on Rule Updates tab
+When user navigates back to Rules Management page
+Then Rules Management Page is properly displayed
+```
+
+#### **Scenario: User can filter prebuilt rules by rule name or by tag**
+
+```Gherkin
+Given a user is on Rule Updates tab
+When user filters by <filter>
+Then Rule Updates tab is properly updated
+
+Examples:
+  | filter                  |
+  | rule name on search bar |
+  | Tag filter              |
+```
```

Comment:
I think this one is too vague so I removed it.

- We should add detailed scenarios to the `Rule upgrade workflow: filtering, sorting, pagination` section.